src/duckling/game/collision/collision.component.ts

```diff
@@ -110,12 +110,7 @@ export class CollisionComponent implements OnDestroy {
 
     private _buildCollisionTypeOptions() : SelectOption[] {
         let selectOptions : SelectOption[] = [];
-        for (let collisionType of this._collisionTypes.collisionTypes.getValue()) {
-            selectOptions.push({
-                value: collisionType,
-                title: collisionType
-            });
-        }
+        this._collisionTypes.collisionTypes.getValue().map(collisionType => selectOptions.push({value: collisionType, title: collisionType}));
```

Comment:
return this._collisionTypes.collisionTypes.getValues().map(collisionType => { value: collisionType, title: collisionType});